pkg/namespace/cmd.go

```diff
@@ -77,14 +77,14 @@ func (c cmd) Modify(ns Namespace, b *Builder) error {
 		}
 		return ns.Mount(servername, old, spec, c.flag)
 	case UNMOUNT:
-		new, old := "", ""
+		newNS, oldNS := "", ""
```

Comment:
I believe generally, for the zero value, 
var newNS, oldNS string 
is preferred?

Reply:
I prefer this style, but not sure if anyone has any consensus about that.